addons/info/src/components/stories/index.js

```diff
@@ -0,0 +1,359 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
```

Comment:
Move examples to the cra-kitchen-sink

Reply:
I don't have examples in this storybook. It's more to have internal components in isolation and to test them with mock data. Do we need all this stuff in cra-kitchen-sink?
https://github.com/storybooks/storybook/pull/1501#issuecomment-325218933

Reply:
Yes i think it's better to put the edge cases of a addon in a story inside the kitchen sink:
```js
storiesOf('addons/info',module).add('usecase of info', () => {});
```